templates/search/noresults.html.tx

```diff
@@ -10,13 +10,14 @@
 %       include 'search/form.tx'
         <div class="grid-row">
              <div class="column-two-thirds">
-                     <article id="no-results" class="text">
+                     <div id="no-results" role="status">
                        <% if $results.searchTerm { %>
                             <h2 class="heading-large" id="no-results-heading" >No results found</h2>
                              <p>
                                  There are no results that match your search. Try searching again using different search criteria.
                              </p>
                        <% } %>
+                     </div>
```

Comment:
Isn't this div tag incorrectly indented?

Reply:
I'll look into this one too.